app/models/tagging.rb

```diff
@@ -1,9 +1,8 @@
 class Tagging < ApplicationRecord
-  belongs_to :tagger, polymorphic: true, inverse_of: :taggings
+  belongs_to :tagger, polymorphic: true, inverse_of: :taggings, autosave: true
```

Comment:
When would we need to auto-save a tag after modifying its taggings? I'm fine with this change, I'm just wondering.

Reply:
Nice, thanks for explaining this!